I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 38, in script
    $ submit_to_google_forms(first_name, last_name, favorite_color)
  File "game/script.rpy", line 38, in <module>
    $ submit_to_google_forms(first_name, last_name, favorite_color)
  File "game/script.rpy", line 4, in submit_to_google_forms
    renpy.emscripten.run_script(js_code)
AttributeError: 'bool' object has no attribute 'run_script'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 38, in script
    $ submit_to_google_forms(first_name, last_name, favorite_color)
  File "C:\Users\antoi\OneDrive\Desktop\renpy\renpy-8.1.0-sdk\renpy\ast.py", line 1138, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\antoi\OneDrive\Desktop\renpy\renpy-8.1.0-sdk\renpy\python.py", line 1122, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/script.rpy", line 38, in <module>
    $ submit_to_google_forms(first_name, last_name, favorite_color)
  File "game/script.rpy", line 4, in submit_to_google_forms
    renpy.emscripten.run_script(js_code)
AttributeError: 'bool' object has no attribute 'run_script'

Windows-10-10.0.22631 AMD64
Ren'Py 8.1.0.23051307
test 1.0
Mon Mar  4 20:57:41 2024
